server/src/main/java/io/crate/metadata/Reference.java

```diff
@@ -73,6 +73,9 @@ static int indexOf(Iterable<? extends Reference> refs, ColumnIdent column) {
     @Nullable
     Symbol defaultExpression();
 
+    @Nullable
+    String formattedDefaultExpression();
```

Comment:
Would it be possible to only keep the `Symbol` version of the default expression? 

Streaming both will unnecessarily bloat the streaming payload for all selects/ inserts/updates involving columns with default expressions

Reply:
https://github.com/crate/crate/pull/14011